### ISSUE TITLE
Deliver support for soverign clouds

### ISSUE THREAD

**Issue by ddyett** [microsoftgraph/microsoft-graph-explorer-v4#1068]

**Comment by ddyett** [comment]
non-committed feature.  Assigning to PM doing investigation on what would need to happen here.

**Comment by lehrt** [comment]
Graph explorer is no longer in active development and has moved to maintenance mode. No more major features like this are funded. If you are an internal Microsoft employee, you may reach out to Graph Explorer team to have an internal discussion about funding new features.
